sktime/registry/_tags.py

```diff
@@ -1826,6 +1826,12 @@ class y_inner_mtype(_BaseTag):
         ("str", ["temporal", "instance"]),
         "whether the splitter splits by time or by instance (panel/hierarchy index)",
     ),
+    (
+        "capability:categorical-in-X",
+        "estimator",
```

Comment:
As per the design proposal, I thought we are not planning to add support for any estimator immediately, only forecasters. Please let me know if it was a misunderstanding, having support for all will obviously be preferable.

Reply:
Yes, support is only for forecasters for now. 
I will change it.